tests/web3signer/key-config.yaml

```diff
@@ -0,0 +1,4 @@
+type: "file-keystore"
+keyType: "SECP256K1"
+keystoreFile: "/data/keystore/opr.ecdsa.key.json"
+keystorePasswordFile: "/data/password/opr.ecdsa.key.txt"
```

Comment:
oh this is also how web3signer works? only reads files?

Reply:
Kinda don't like this for tests, because then you need to mount a bunch of files in the docker container, kinda ugly.

Reply:
this is one of the way it works - like local keystore but in the web3 signer instance - in order to be predictable I have to mount it and make sure I have key loaded

the other way where you can have AWS , azure, hashicorp as backend don't work this way. so this is only for testing